tests/testFindAstrometryNetDataDir.py

```diff
@@ -29,7 +29,7 @@ def setupAstrometryNetDataDir(name, rootDir=None, verbose=False):
 class TestAstrometryNetDataDirDiscovery(unittest.TestCase):
 
     def test_photocal(self):
-        datapath = setupAstrometryNetDataDir('photocal')
+        datapath = setupAstrometryNetDataDir('t2155')
```

Comment:
This isn't a build tag, it's the name of a directory inside tests/astrometry_net_data. No other test was using the a.net version of "photocal" and this one test didn't care which set of a.net data it used, so I switched it to use the other available a.net data and discarded the now-unused "photocal" data.